### ISSUE TITLE
Bor mainnet stuck

### ISSUE THREAD

**Issue by masayil** [ledgerwatch/erigon#5272]
version: v2022.08.03
![image](https://user-images.githubusercontent.com/65291464/188349281-71794741-80a6-4053-829a-4498ca36c76a.png)

**Comment by AskAlexSharov** [comment]
logs | grep -i err
logs | grep -i warn

**Comment by masayil** [comment]
![image](https://user-images.githubusercontent.com/65291464/188349595-8445c560-bec3-4ad9-ae2c-f88ea5456f96.png)

**Comment by AskAlexSharov** [comment]
I think It's fixed in `devel`

Try switch to `devel` and do:
```
make integration
./build/bin/integration state_stages    --unwind=1000 --chain=bor-mainnet 
./build/bin/integration stage_headers --unwind=1000 --chain=bor-mainnet 

// then start erigon as usually
```

**Comment by masayil** [comment]
> I think It's fixed in `devel`
> 
> Try switch to `devel` and do:
> 
> ```
> make integration
> ./build/bin/integration state_stages    --unwind=1000 --chain=bor-mainnet 
> ./build/bin/integration stage_headers --unwind=1000 --chain=bor-mainnet 
> 
> // then start erigon as usually
> ```

I will try it.

**Comment by masayil** [comment]
1.`./build/bin/integration state_stages    --unwind=1000 --chain=bor-mainnet ` done~!
2. `./build/bin/integration stage_headers --unwind=1000 --chain=bor-mainnet ` done~!
3. Inserting headers   doing~
![image](https://user-images.githubusercontent.com/65291464/188381918-432a6386-b24b-4fa4-a90d-795f57c14d7d.png)

**Comment by masayil** [comment]
> I think It's fixed in `devel`
> 
> Try switch to `devel` and do:
> 
> ```
> make integration
> ./build/bin/integration state_stages    --unwind=1000 --chain=bor-mainnet 
> ./build/bin/integration stage_headers --unwind=1000 --chain=bor-mainnet 
> 
> // then start erigon as usually
> ```

![image](https://user-images.githubusercontent.com/65291464/188405397-8b4fc102-8140-4205-b934-0d51a3f166c1.png)

**Comment by masayil** [comment]
It doesn't work.I try to resync the node

**Comment by enriavil1** [comment]
I just fixed that issue in devel with this PR 

> > I think It's fixed in `devel`
> > Try switch to `devel` and do:
> > ```
> > make integration
> > ./build/bin/integration state_stages    --unwind=1000 --chain=bor-mainnet 
> > ./build/bin/integration stage_headers --unwind=1000 --chain=bor-mainnet 
> > 
> > // then start erigon as usually
> > ```
> 
> ![image](https://user-images.githubusercontent.com/65291464/188405397-8b4fc102-8140-4205-b934-0d51a3f166c1.png)

I just fixed that issue in devel with this PR #5283
